src/script/components/panel/SingleAction/SingleAction.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+
+import {Button, ButtonVariant, FlexBox} from '@wireapp/react-ui-kit';
+
+import {t} from 'Util/LocalizerUtil';
+
+export interface MenuItem {
+  click: () => void;
+  icon: string;
+  identifier: string;
+  label: string;
+}
+
+export interface SingleActionProps {
+  item: MenuItem;
+  onCancel: (action: any) => void;
+}
+
+const SingleAction: React.FC<SingleActionProps> = ({item, onCancel}) => {
```

Comment:
If I good remember we shouldn't use FC<T>, instead of this do this:

```suggestion
const SingleAction = ({item, onCancel}: SingleActionProps) => {
```